authentication/configuring-internal-oauth.adoc

```diff
@@ -4,6 +4,7 @@ include::modules/common-attributes.adoc[]
 :context: configuring-internal-oauth
 toc::[]
 
+// TODO: What on earth is this?
 [IMPORTANT]
 ====
 Configuring these options must change because they're set in the master
```

Comment:
@kalexand-rh Any chance you remember what this statement is/was supposed to mean? Looks like it was added here: https://github.com/openshift/openshift-docs/pull/13051/files#diff-9b330ab3e6f94100d5ff06d0a7a83d6cR8-R12. But I can't track down any further info on it from the related Jiras.

Almost sounds like some internal note that shouldn't have been rendered, but I'm not sure. Let me know if you think it can be removed or if there's a reason we need it.

Reply:
I'm going to remove this note. I think @kalexand-rh is out for a bit so we can always add it back in if she confirms that it's needed. @rhpmali Let me know if you have any thoughts on this note or objections to its removal.